src/Init/Data/Bool.lean

```diff
@@ -220,6 +220,14 @@ def toNat (b:Bool) : Nat := cond b 1 0
 theorem toNat_le_one (c:Bool) : c.toNat ≤ 1 := by
   cases c <;> trivial
 
+theorem toNat_lt (b : Bool) : b.toNat < 2 :=
+  Nat.lt_succ_of_le (toNat_le_one _)
```

Comment:
I would prefer consistent names here, but I think `toNat_lt` is correct, so will rename the other one.